cli/tests/test_working_copy.rs

```diff
@@ -20,13 +20,18 @@ fn test_snapshot_large_file() {
     test_env.jj_cmd_ok(test_env.env_root(), &["init", "repo", "--git"]);
     let repo_path = test_env.env_root().join("repo");
 
-    test_env.add_config(r#"snapshot.max-new-file-size = "10""#);
+    test_env.add_config(r#"snapshot.max-new-file-size = 10"#);
```

Comment:
Could you add a test that does "10B". "10KB" would be even better, but maybe that's too much hassle.